wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java

```diff
@@ -154,6 +154,7 @@ public WikiServiceImpl(ConfigurationManager configManager,
 
   @Override
   public void start() {
+    initTemplates();

```

Comment:
Thanks Thomas, the root cause comes from this commit https://github.com/exodev/wiki/commit/8b0552651ab4eb029f54837f86b161a8a9a45f98.